package.json

```diff
@@ -79,7 +79,8 @@
       "binaries": [
         "./mono.linux-x86",
         "./run"
-      ]
+      ],
+      "installTestPath": "./bin/mono.linux-x86"
```

Comment:
This is a little unfortunate. Kind of wish we didn't have to specially list this for every dependency.